api/app/views/spree/api/shipments/big.v1.rabl

```diff
@@ -41,6 +41,13 @@ child order: :order do
 
   child payments: :payments do
     attributes :id, :amount, :display_amount, :state
+    child source: :source do |s|
+      if s.is_a?(Spree::CreditCard)
```

Comment:
What about switching this to:

```
if s.respond_to?(:cc_type)
```

That will allow other objects which implement cc_type to use this. Probably not super helpful with this attribute, but it's a good pattern to follow.